config/webhook/patches/namespace.yaml

```diff
@@ -1,3 +1,6 @@
 - op: replace
   path: /webhooks/0/clientConfig/service/namespace
   value: kfp-operator-system
+- op: replace
+  path: /webhooks/1/clientConfig/service/namespace
+  value: kfp-operator-system
```

Comment:
There must be a better way, even with awful Kustomize! I just couldn't find it in a few hours of trying 😮‍💨